packages/core/contracts/financial-templates/common/FinancialContractInterface.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "../../common/implementation/FixedPoint.sol";
+
+/**
+ * @notice External methods that any financial contract must implement.
+ */
+interface FinancialContractInterface {
```

Comment:
+1 will implement

Reply:
OOC why isn't thi salready in AdminstrateeInterface?

Reply:
What about just adding `pfc()` to the PerpetualContractInterface versus adding `pfc` to AdministrateeInterface?

Reply:
Actually ^ is problematic because the ExpiringMultiParty would have to implement `withdrawFundingRateFees` because `FeePayer` would have to be a `PerpetualInterface`

Reply:
Added `pfc` to `AdministrateeInterface` in this [PR](https://github.com/UMAprotocol/protocol/pull/2168/commits/6c5738091a74d24c92724d4a7d470ebf07e5e6a8)

Reply:
LGTM! I actually liked your naming here. Maybe not in this PR, but we might consider changing the AdministrateeInterface to FinancialContractInterface, as I think that name is a little clearer.